packages/go_router/lib/src/router.dart

```diff
@@ -299,6 +302,15 @@ class GoRouter implements RouterConfig<RouteMatchList> {
   @override
   late final GoRouteInformationParser routeInformationParser;
 
+  /// When set to [true], the route returned by `getInitialRoute` will take precedence over the
+  /// platform's default initial location. This allows developers to control the starting route
+  /// of the application independently of the platform.
+  ///
+  /// Setting this parameter to [false] (default) will allow the platform's default initial
+  /// location to be used. It's advisable to only set this to true if you explicitly want to
```

Comment:
```suggestion
  /// location to be used even if the `initialLocation` is set. It's advisable to only set this to true if one explicitly want to
```

Reply:
our style guide suggest doc string should avoid using `I`, `we`, `you`.

Reply:
I went over that, it's fixed